source/content/drush-versions.md

```diff
@@ -6,7 +6,9 @@ categories: [drupal,workflow,platform,develop]
 ---
 By default, Pantheon runs Drush 8 on newly created Drupal sites. Drush 8 is compatible with Drupal 6, 7, and 8.
 
-The Drush version is set when the site is created. Sites created before Nov 4, 2015 (and Drupal 8 sites created before Sep 3, 2015) use an older version of Drush that is incompatible with later versions of PHP. You can [check your current Drush version via Terminus](/drush-versions/#verify-current-drush-version). Updating Drush can be accomplished by [a simple addition to your pantheon.yml file](/drush-versions/#configure-drush-version) below.
+The Drush version is set when the site is created. Sites created before Nov 4, 2015 (and Drupal 8 sites created before Sep 3, 2015) use an older version of Drush that is incompatible with later versions of PHP. You can [check your current Drush version via Terminus](#verify-current-drush-version). Updating Drush can be accomplished by [a simple addition to your pantheon.yml file](#configure-drush-version) below.
+
+Drush 10 is available for [site-local installation](#site-local-drush-usage). It requires Drupal 8 or newer, [Composer](/composer/), and PHP 7.1 or newer.
```

Comment:
It is also possible to select Drush 10 via pantheon.yml. This works fine for Drupal 8.

I'm not sure that we need to document Drupal 9 in this document at this time. If we do, perhaps we could have a footnote somewhere that pre-release versions of Drupal 9 (not supported) currently require Drush 10 + a site-local install using Composer.